api/src/test/java/org/pmiops/workbench/db/dao/UserDaoTest.java

```diff
@@ -8,7 +8,6 @@
 import java.sql.Timestamp;
 import java.time.Instant;
 import java.util.List;
-import javax.jdo.annotations.Transactional;
```

Comment:
not sure why these package upgrades caused this to fail, but it's the wrong import regardless